packages/strapi-admin/admin/src/components/Users/LoginModalSection/index.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ModalSection } from 'strapi-helper-plugin';
+import { Padded } from '@buffetjs/core';
+import { Row } from 'reactstrap';
+import { useIntl } from 'react-intl';
+import loginSettingsForm from 'ee_else_ce/components/Users/ModalCreateBody/utils/loginSettingsForm';
```

Comment:
I'm not sure to understand how it works: if EE is activated, we import the CE one and otherwise, we import the EE one? Am I right?

Reply:
Yes 😄